commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -297,7 +297,7 @@
                 } else {
                     $scope.rollback = false;
                 }
-            } else if (newValue !== undefined && newValue.name === "") {
+            } else if (newValue && newValue.name === "") {
```

Comment:
While at it, can we change this to (newValue && newValue.name) ? I hate these === "", they just cause issues